src/routes/docs/advanced/security/+layout.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+	import Docs from '$lib/layouts/Docs.svelte';
+	import Sidebar, { type NavParent, type NavTree } from '$lib/layouts/Sidebar.svelte';
+
+	const parent: NavParent = {
+		href: '/docs',
+		label: 'Security'
+	};
+
+	const navigation: NavTree = [
+		{
+			label: 'Security',
+			items: [
+				{
+					label: 'Getting started',
```

Comment:
Will calling this `Overview` break any existing conventions?

Reply:
No this will following existing conventions LOL